core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -1000,6 +1085,9 @@ final class Stream[+F[_], +O] private (private val free: FreeC[Algebra[Nothing,
   /**
     * Alias for `map(f).foldMonoid`.
     *
+    * @returns A singleton stream whose value is the result of the fold, if `this` is a finite stream.
```

Comment:
Might be better to elide this one as the docs for foldMonoid have the full details.